test/server/cardImplementations/01_SOR/MonMothmaVoiceoftheRebellion.spec.js

```diff
@@ -0,0 +1,120 @@
+describe('Mon Mothma, Voice of the Rebellion', function() {
+    integration(function() {
+        describe('Mon Mothma\'s Ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mon-mothma#voice-of-the-rebellion'],
+                        deck: ['cell-block-guard', 'pyke-sentinel', 'volunteer-soldier', 'cartel-spacer', 'battlefield-marine', 'wampa', 'viper-probe-droid', 'snowtrooper-lieutenant'],
+                        deckSize: 8,
+                        leader: ['leia-organa#alliance-general'],
+                        resources: ['atst', 'atst', 'atst', 'atst', 'atst', 'atst']
+                    }
+                });
+
+                this.monMothma = this.player1.findCardByName('mon-mothma#voice-of-the-rebellion');
+                this.battlefieldMarine = this.player1.findCardByName('battlefield-marine');
+
+                this.cartelSpacer = this.player1.findCardByName('cartel-spacer');
+                this.cellBlockGuard = this.player1.findCardByName('cell-block-guard');
+                this.pykeSentinel = this.player1.findCardByName('pyke-sentinel');
+                this.volunteerSoldier = this.player1.findCardByName('volunteer-soldier');
+
+                this.noMoreActions();
+            });
+
+            it('should prompt to choose a Rebel from the top 5 cards', function () {
+                this.player1.clickCard(this.monMothma);
+                expect(this.player1).toHavePrompt('Select a card to reveal');
+                expect(this.player1).toHaveDisabledPromptButton(this.cellBlockGuard.title);
+                expect(this.player1).toHaveDisabledPromptButton(this.volunteerSoldier.title);
+                expect(this.player1).toHaveDisabledPromptButton(this.pykeSentinel.title);
+                expect(this.player1).toHaveDisabledPromptButton(this.cartelSpacer.title);
+                expect(this.player1).toHavePromptButton(this.battlefieldMarine.title);
+                expect(this.player1).toHavePromptButton('Take nothing');
+            });
+
+            it('should reveal the chosen Rebel', function() {
```

Comment:
I'm not sure if it's the best test practice but it might be okay to stitch the first 4 test cases here into one big test. The reason being that (a) each individual test case has to go through a lot of setup and the time starts to add up across tests, and (b) these are really testing different aspects of the same code action (play Mon Mothma and select Marine). So it feels okay to me to combine them, but I'm fine if you want to keep them as-is for cleanliness. Ditto for Tarkin's tests.